1.24.2

matteobruni opened the pull request:
feat: added mass to particle, this will improve collisions

matteobruni committed:
feat: added mass to particle, this will improve collisions

matteobruni committed:
build: improved some imports

ghost commented:
## DeepCode's analysis on [#9e4846](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/f56f9d1abb1851eeb237fffd52d4d0ccf80a44a2/matteobruni/tsparticles/9e48460708f662d95cdce01a5d13dd1b1dd30db8/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/9e48460708f662d95cdce01a5d13dd1b1dd30db8/core/main/src/Core/Particle.ts#L42">Particle.ts:42</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/9e48460708f662d95cdce01a5d13dd1b1dd30db8/core/main/src/Core/Particle.ts#L43">Particle.ts:43</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/9e48460708f662d95cdce01a5d13dd1b1dd30db8/core/main/src/Core/Particle.ts#L44">Particle.ts:44</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/f56f9d1abb1851eeb237fffd52d4d0ccf80a44a2/matteobruni/tsparticles/9e48460708f662d95cdce01a5d13dd1b1dd30db8/pr/_/%2Fcore%2Fmain%2Fsrc%2FCore%2FParticle.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/f56f9d1abb1851eeb237fffd52d4d0ccf80a44a2/matteobruni/tsparticles/9e48460708f662d95cdce01a5d13dd1b1dd30db8/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
build: added docs to main demo

matteobruni committed:
build: fixed samples url

matteobruni committed:
chore(release): published new version

 - ng-particles@2.7.2
 - inferno-particles@1.7.2
 - jquery-particles@1.24.2
 - preact-particles@1.24.2
 - react-tsparticles@1.24.3
 - svelte-particles@1.7.2
 - particles.vue@2.7.2
 - particles.vue3@1.7.2
 - tsparticles-editor@1.24.2
 - tsparticles@1.24.2
 - ng-particles-demo@1.9.2
 - editor-demo@1.9.2
 - inferno-particles-demo@1.7.2
 - jquery-particles-demo@1.9.2
 - tsparticles-demo@1.9.2
 - preact-particles-demo@1.9.2
 - react-tsparticles-demo@1.9.3
 - svelte-demo@1.7.2
 - particles.vue-demo@1.9.2
 - particles.vue3-demo@1.7.2
 - tsparticles-preset-60fps@1.9.2
 - tsparticles-preset-background-mask@1.9.2
 - tsparticles-preset-basic@1.9.2
 - tsparticles-preset-big-circles@1.9.2
 - tsparticles-preset-bouncing@1.9.2
 - tsparticles-preset-bubbles@1.9.2
 - tsparticles-preset-fire@1.9.2
 - tsparticles-preset-font-awesome@1.9.2
 - tsparticles-preset-fountain@1.9.2
 - tsparticles-preset-sea-anemone@1.9.2
 - tsparticles-preset-snow@1.9.2
 - tsparticles-preset-stars@1.9.2
 - tsparticles-shape-bubble@1.9.2
 - tsparticles-shape-confetti@1.9.2
 - tsparticles-shape-heart@1.9.2
 - tsparticles-shape-multiline-text@1.9.2
 - tsparticles-shape-rounded-rect@1.9.2
 - tsparticles-shape-spiral@1.9.2
 - cra-template-particles-typescript@1.7.3
 - cra-template-particles@1.7.3
 - tsparticles-website@1.24.2

matteobruni committed:
build: updated website
